app/src/main/res/layout/activity_main.xml

```diff
@@ -137,6 +137,13 @@
                         android:layout_height="match_parent"
                         android:layout_weight="1"
                         app:srcCompat="@drawable/ic_baseline_photo_size_select_large_24" />
+
+                    <ImageButton
```

Comment:
Could this be implemented as a toggle button (like the crop/lock actions are)?

Also I would prefer the button to be on the bottom toolbar, but if there's no room for it anymore then it's fine.